src/components/package-show.js

```diff
@@ -22,100 +20,96 @@ export default function PackageShow({ vendorPackage, packageTitles }, { intl })
+                {formatISODateWithoutTime(packageRecord.customCoverage.beginCoverage)} - {formatISODateWithoutTime(packageRecord.customCoverage.endCoverage)}
+              </div>
+            </KeyValueLabel>
+          )}
 
+          <hr />
+
+          <KeyValueLabel label="Selected">
+            <div data-test-eholdings-package-details-selected>
+              {packageRecord.isSelected ? 'Yes' : 'No'}
+            </div>
+          </KeyValueLabel>
+
+          <hr />
+
+          {packageRecord.visibilityData.isHidden && (
+            <div data-test-eholdings-package-details-is-hidden>
+              <p><strong>This package is hidden.</strong></p>
+              <p><em>{packageRecord.visibilityData.reason}</em></p>
               <hr />
+            </div>
+          )}
 
-              {packageRecord.visibilityData.isHidden && (
-                <div data-test-eholdings-package-details-is-hidden>
-                  <p><strong>This package is hidden.</strong></p>
-                  <p><em>{packageRecord.visibilityData.reason}</em></p>
-                  <hr />
-                </div>
-              )}
-
-              {packageTitles.isResolved && packageTitles.content.length ? (
-                <div>
-                  <h3>Titles</h3>
-                  <List data-test-eholdings-package-details-title-list>
-                    {packageTitles.content.map(item => (
-                      <TitleListItem
-                        key={item.titleId}
-                        item={item}
-                        link={`/eholdings/vendors/${packageRecord.vendorId}/packages/${packageRecord.packageId}/titles/${item.titleId}`}
-                        showSelected
-                      />
-                    ))}
-                  </List>
-                </div>
-              ) : packageTitles.isResolved ? (
-                <p>No Titles Found</p>
-              ) : (
-                <Icon icon="spinner-ellipsis" />
-              )}
+          {packageTitles.isResolved && packageTitles.content.length ? (
+            <div>
+              <h3>Titles</h3>
+              <List data-test-eholdings-package-details-title-list>
+                {packageTitles.content.map(item => (
+                  <TitleListItem
+                    key={item.titleId}
+                    item={item}
+                    link={`/eholdings/vendors/${packageRecord.vendorId}/packages/${packageRecord.packageId}/titles/${item.titleId}`}
+                    showSelected
+                  />
+                ))}
+              </List>
             </div>
-          ) : vendorPackage.isRejected ? (
-            <p data-test-eholdings-package-details-error>
-              {vendorPackage.error.length ? vendorPackage.error[0].message : vendorPackage.error.message}
-            </p>
+          ) : packageTitles.isResolved ? (
+            <p>No Titles Found</p>
           ) : (
-            <Icon icon='spinner-ellipsis' />
+            <Icon icon="spinner-ellipsis" />
```

Comment:
how come we switched to double quotes here, but we're using single quotes on line 111?  i could've sworn ESLint enforced one or the other.  it's obviously not the end of the world if we aren't consistent in our quoting style, it's just puzzling

Reply:
That is strange that eslint didn't pick this up. I usually go with double quotes for JSX fwiw.

Reply:
Which is it?